Bug: Module 'django.contrib.admindocs' has no attribute 'urls'

Source:

```python
from django.contrib.admindocs import urls as admindocs_urls
```


```
server/urls.py:16: error: Module 'django.contrib.admindocs' has no attribute 'urls'
```

Ref: https://github.com/wemake-services/wemake-django-template/blob/81f162b8ce26671429c32dfa2701648c8ddcb038/%7B%7Bcookiecutter.project_name%7D%7D/server/urls.py#L16
Real source: https://github.com/django/django/blob/master/django/contrib/admindocs/urls.py

--- a/django-stubs/contrib/admindocs/urls.pyi
+++ b/django-stubs/contrib/admindocs/urls.pyi
@@ -1,0 +1,3 @@
+from typing import Any, List
+
+urlpatterns: List[Any] = ...

--- a/django-stubs/contrib/flatpages/urls.pyi
+++ b/django-stubs/contrib/flatpages/urls.pyi
@@ -1,0 +1,3 @@
+from typing import Any, List
+
+urlpatterns: List[Any] = ...

--- a/django-stubs/contrib/staticfiles/urls.pyi
+++ b/django-stubs/contrib/staticfiles/urls.pyi
@@ -2,6 +2,6 @@ from typing import Any, List, Optional
 
 from django.urls.resolvers import URLPattern
 
-urlpatterns: Any
+urlpatterns: List[Any] = ...
 
 def staticfiles_urlpatterns(prefix: Optional[str] = ...) -> List[URLPattern]: ...

